validator/rpc/accounts.go

```diff
@@ -148,3 +149,28 @@ func (s *Server) BackupAccounts(
 		ZipFile: buf.Bytes(),
 	}, nil
 }
+
+// DeleteAccounts deletes accounts from a user if their wallet is an imported wallet.
+func (s *Server) DeleteAccounts(
+	ctx context.Context, req *pb.DeleteAccountsRequest,
+) (*pb.DeleteAccountsResponse, error) {
+	if req.DeletePublicKeys == nil || len(req.DeletePublicKeys) < 1 {
```

Comment:
It would be good to also add a test for this precondition.

Reply:
added.